### PR TITLE
items: add temporary location field

### PR DESCRIPTION
**Pull request by Garfield-fr**
* Displays the `temporary_location` field according to the
  exhibitions/courses and temporary location cascade on the document
  detailed view of the public interface.
* Displays the `temporary_location` field on the item detailed view of the
  professional interface.

* Closes rero/rero-ils#2038.

Co-Authored-by: Bertrand Zuchuat <bertrand.zuchuat@rero.ch>

## Code review check list

- [ ] Commit message template compliance.
- [ ] Commit message without typos.
- [ ] File names.
- [ ] Functions names.
- [ ] Functions docstrings.
- [ ] Unnecessary commited files?
- [ ] Cypress tests successful?

### CODE REVIEW

**Review by lauren-d — APPROVED** [review]

File: projects/admin/src/app/record/detail-view/document-detail-view/holding/default-holding-item/default-holding-item.component.html [hunk 1]
```diff
@@ -64,10 +64,24 @@
...
+        <!-- ITEM IN COLLECTION -->
+        <ng-container *ngIf="item.metadata.pid | itemInCollection | async as collections">
+          <div class="col-4 pl-5 font-weight-bold label-title" translate>Exhibition/course</div>
+          <div class="col-8">
+            <ng-container *ngFor="let collection of collections; let last=last">
+              <a [routerLink]="['/records', 'collections', 'detail', collection.metadata.pid]">{{ collection.metadata.title }}</a>
+              {{ last ? '' : '; ' }}
+            </ng-container>
+          </div>
+        </ng-container>
```
Comment (zannkukai) [inline projects/admin/src/app/record/detail-view/document-detail-view/holding/default-holding-item/default-holding-item.component.html:81]: If you item has a `temporary_location` and belongs to one/more `collections`, you display both informations. Right ? 
@pronguen Is it the correct use-case ? I understood that we : 
1) display collection informations if item belong to a collection
2) **if not** and item has a `temporary_location`, then display this information.
3) **if not** display nothing

File: projects/admin/src/app/record/detail-view/item-detail-view/item-detail-view.component.html [hunk 2]
```diff
@@ -78,6 +78,21 @@ <h1>{{ 'Barcode' | translate }} {{ record.metadata.barcode }}</h1>
...
+      <!-- ITEM IN COLLECTION -->
+      <ng-container *ngIf="record.metadata.pid | itemInCollection | async as collections">
+        <dt class="col-3 label-title" translate>Exhibition/course</dt>
+        <dd class="col-9">
+          <ng-container *ngFor="let collection of collections; let last=last">
+            <a [routerLink]="['/records', 'collections', 'detail', collection.metadata.pid]">{{ collection.metadata.title }}</a>
+            {{ last ? '' : '; ' }}
+          </ng-container>
+        </dd>
+      </ng-container>
```
Comment (zannkukai) [inline projects/admin/src/app/record/detail-view/item-detail-view/item-detail-view.component.html:95]: same remark

File: projects/admin/src/app/record/detail-view/document-detail-view/holding/default-holding-item/default-holding-item.component.html (hunk 1 above)
Comment (pronguen) [reply projects/admin/src/app/record/detail-view/document-detail-view/holding/default-holding-item/default-holding-item.component.html:81]: @zannkukai Bertrand told me he implemented a "cascade" for the public view: so collection, else temporary_location, else nothing.
For the pro interface, we finally decided to display both with the real label.

**Review by zannkukai — APPROVED** [review]

**Review by iGormilhit — CHANGES_REQUESTED** [review]
Commit message proposition:

```
items: add temporary location field

* Displays the `temporary_location` field according to the
  exhibitions/courses and temporary location cascade on the document
  detailed view of the public interface.
* Displays the `temporary_location` field on the item detailed view of the
  professional interface.

* Closes rero/rero-ils#2038.
```

**Review by iGormilhit — APPROVED** [review]

**Pull request merged**